pkg/kwokctl/cmd/get/get.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package get
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kwok/pkg/kwokctl/cmd/get/artifacts"
+	"sigs.k8s.io/kwok/pkg/kwokctl/cmd/get/clusters"
+	"sigs.k8s.io/kwok/pkg/kwokctl/cmd/get/kubeconfig"
+	"sigs.k8s.io/kwok/pkg/logger"
+)
+
+// NewCommand returns a new cobra.Command for get
+func NewCommand(logger logger.Logger) *cobra.Command {
+	cmd := &cobra.Command{
+		Args:  cobra.NoArgs,
+		Use:   "get",
+		Short: "Gets one of [artifacts, clusters, kubeconfig]",
+		Long:  "Gets one of [artifacts, clusters, kubeconfig]",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return fmt.Errorf("subcommand is required")
+		},
+	}
+	// add subcommands
+	cmd.AddCommand(clusters.NewCommand(logger))
```

Comment:
seems `kwokctl get artifacts --name kwok` returns the default images (instead of binaries) that the cluster is using:

```
⇒  kwokctl get artifacts --name kwok
https://dl.k8s.io/release/v1.24.2/bin/darwin/arm64/kubectl
registry.k8s.io/etcd:3.5.4-0
registry.k8s.io/kube-apiserver:v1.24.2
registry.k8s.io/kube-controller-manager:v1.24.2
registry.k8s.io/kube-scheduler:v1.24.2
registry.k8s.io/kwok/kwok:v0.1.0
docker.io/prom/prometheus:v2.35.0
```

The command I used to spin up the cluster was:

```
kwokctl create cluster --runtime binary --kube-scheduler-binary /usr/local/bin/kube-scheduler@1.24.3 --kube-controller-manager-binary /usr/local/bin/kube-controller-manager@1.24.3  --kube-apiserver-binary /usr/local/bin/kube-apiserver@1.24.3 --kwok-controller-binary /usr/local/bin/kwok --etcd-binary /opt/homebrew/bin/etcd
```

Reply:
Fixed